shell/common/null_platform_view.cc

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/common/null_platform_view.h"
+
+#include "flutter/shell/common/null_rasterizer.h"
+#include "flutter/shell/common/shell.h"
+
+namespace shell {
+
+NullPlatformView::NullPlatformView()
+    : PlatformView(std::unique_ptr<Rasterizer>(new NullRasterizer())),
+      weak_factory_(this) {}
+
+void NullPlatformView::Attach() {
+  CreateEngine();
+  PostAddToShellTask();
+}
+
+NullPlatformView::~NullPlatformView() = default;
+
+fxl::WeakPtr<NullPlatformView> NullPlatformView::GetWeakPtr() {
+  return weak_factory_.GetWeakPtr();
+}
+
+bool NullPlatformView::ResourceContextMakeCurrent() {
+  return false;
+}
+
+void NullPlatformView::RunFromSource(const std::string& assets_directory,
+                                     const std::string& main,
+                                     const std::string& packages) {}
```

Comment:
A comment here saying that hot-reload of the null platform view is unsupported would be great.

Reply:
Done